### PR TITLE
Donation page #16

### PR DESCRIPTION
**Pull request by Tjs002**
Added Donation Page

### CODE REVIEW

**Review by TheLastProject — CHANGES_REQUESTED** [review]
I strongly suggest you follow a markdown tutorial. HTML tags should only be used for features Markdown doesn't support, but you used them everywhere, even for features that Markdown does support.

Also, you added a lot of extra style attributes, causing this page to look completely different from all the other pages and ignore the global website theming. If this page's design would be unique, that would be understandable, but it isn't unique in any way, so now it just has a different font size and style than the rest of the website for... no real reason?

The general structure is fine though, so let's start with removing all of the HTML tags where possible (really, almost everything here can be Markdown, even the donate buttons!). And then we can look at the text so it's not a one-on-one copy of the KISS website :)

File: _pages/donate.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+<p style="font-size: 2.3rem;font-family:Merriweather, serif;">
+While Catima also has very little costs (currently only the catima.app domain), 
+   Catima also doesn't have any third-party contributors who want to be paid
+   (yet? who knows what the future brings) and it definitely doesn't relate to third-world country stuff 
+   (although I do try to keep Catima lean-and-green and working well on old devices). 
+   Donations for Catima are more a "thank you" than anything else, 
+   so it would be important for people to not think that their money will be used to do something great 
+   (if we do ever get contributors in the future who want to be paid and will create good enough stuff that it's worth it, 
+   I can update the text myself)
+</p>
```
[the remark this answers is not in the document]
Comment (Tjs002) [reply _pages/donate.md:94]: The text is just for review can change it after the review and let me know what text should be used.

File: _pages/donate.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+---
+layout: page
+title: Donate
+include_in_header: false
+include_in_footer: true
+---
+
+# Donate to CATIMA
```
[the remark this answers is not in the document]
Comment (Tjs002) [reply _pages/donate.md:8]: ok i will change the tags to the markdown. let me know how to create buttons in markdown

**Comment by obfusk** [comment]
I'm sorry, I didn't have time this week but I'm hoping to have time to review and help improve the wording etc. a bit next week. Thanks for the help!

**Comment by Tjs002** [comment]
> I'm sorry, I didn't have time this week but I'm hoping to have time to review and help improve the wording etc. a bit next week. Thanks for the help!

Yaa no worries...Let me know if i can assist for further help

**Pull request closed without merging**